python/lsst/pipe/analysis/utils.py

```diff
@@ -1033,3 +1091,188 @@ def getDataExistsRefList(dataRefList, dataset):
     if dataExistsRefList is None:
         raise RuntimeError("dataExistsRef list is empty")
     return dataExistsRefList
+
+
+def fLinear(p, x):
+    return p[0] + p[1]*x
+
+
+def fQuadratic(p, x):
+    return p[0] + p[1]*x + p[2]*x**2
+
+
+def fCubic(p, x):
+    return p[0] + p[1]*x + p[2]*x**2 + p[3]*x**3
+
+
+def orthogonalRegression(x, y, order, initialGuess=None):
+    """Perform an Orthogonal Distance Regression on the given data
+
+    Parameters:
+    ----------
+    x, y : `array`
+       Arrays of x and y data to fit
+
+    Returns:
+    -------
+    `list` of fit parameters (highest power first to mimic np.polyfit return)
+    """
+    if initialGuess is None:
+        linReg= scipyStats.linregress(x, y)
+        initialGuess = [linReg[0], linReg[1]]
+        for i in range(order - 1):
+            initialGuess.insert(0, 0.0)
```

Comment:
An inline comment might help clarify that (and why) you're  padding so that initialGuess is of length order+1,

Reply:
I also added docs for the parameters I managed to skip on the first round!